Std/Data/List/Pairwise.lean

```diff
@@ -213,7 +213,7 @@ theorem map_get_sublist {l : List α} {is : List (Fin l.length)} (h : is.Pairwis
     simp; cases hl'
     have := IH h.of_cons (hd+1) _ rfl (pairwise_cons.mp h).1
     specialize his hd (.head _)
-    have := get_cons_drop .. ▸ this.cons₂ (get l hd)
+    have := (drop_eq_get_cons ..).symm ▸ this.cons₂ (get l hd)
```

Comment:
is the symm here necessary? `▸` rewrites in both directions so `symm` is usually inferrable

Reply:
It fails without `symm`.